app/assets/stylesheets/components/header.scss

```diff
@@ -120,18 +122,23 @@
     opacity: var(--dropdown-opacity, 0);
     visibility: var(--dropdown-visibility, hidden);
   }
-
   &.showing,
-  &.desktop:hover,
-  &.desktop:focus {
+  &.desktop:hover {
     --dropdown-transform: translateY(0) rotateX(0);
     --dropdown-delay: 0;
     --dropdown-opacity: 1;
     --dropdown-visibility: visible;
 
     .crayons-header__menu__trigger {
+      border: 2px solid var(--header-bg-hover);
       color: var(--header-icons-color-hover);
-      background: var(--header-bg-hover);
+    }
+  }
+  &.showing :focus,
+  :focus {
+    &.crayons-header__menu__trigger {
+      border: 2px solid var(--header-button-focus-color);
```

Comment:
One thing I don't love is the flash of the focus style as a mouse user, where the navigation button is focused for a hot second before being moved to the first nav link (for focus management). With `focus-visible` in use, you don't see the focus style being moved to the first nav link as a mouse user. I could try to play with it some more, if anyone feels strongly about it.

Reply:
I've noticed that too - and seemed a bit jarring. In addition - the dropdown now requires me to explicitly open it when navigating with keyboard - which wasn't a case in previously.. Is that intentional?

Reply:
>  the dropdown now requires me to explicitly open it when navigating with keyboard - which wasn't a case in previously.. Is that intentional?

Yes, it is intentional as the dropdown forced keyboard users through it before (quite a few extra tab stops if you're trying to go past). The keyboard navigation convention for dropdowns is to open on Enter or Space key on the triggering element, and make the child items only focusable when the dropdown is open.

Reply:
I pushed an update, should be good now with no flash of the focus outline on click.